spec/install_spec.rb

```diff
@@ -1,26 +1,38 @@
 
```

Comment:
please excuse my torturing of ruby ; I am sure this could be much prettier ... I am happy to be told bluntly how to improve this.